superset/views/dashboard/views.py

```diff
@@ -119,6 +119,32 @@ class DashboardModelViewAsync(DashboardModelView):  # pylint: disable=too-many-a
     route_base = "/dashboardasync"
     include_route_methods = {RouteMethod.API_READ}
 
+    class_permission_name = "Dashboard"
+
```

Comment:
I did not added the class_previous name, and the previous perm dict, because was not aware about that, and the PR from which I inspired and the docs example like

```
An example for compressing permissions using MVC Model Views:

class OneView(ModelView):
    datamodel = SQLAInterface(Contact)
    class_permission_name = "view"
    method_permission_name = {
        'add': 'write',
        'delete': 'write',
        'download': 'write',
        'edit': 'write',
        'list': 'read',
        'muldelete': 'write',
        'show': 'read',
        'api': 'read',
        'api_column_add': 'write',
        'api_column_edit': 'write',
        'api_create': 'write',
        'api_delete': 'write',
        'api_get': 'read',
        'api_read': 'read',
        'api_readvalues': 'read',
        'api_update': 'write'
    }
```

were also the same.